### DISCUSSION TITLE
v0.1.1-beta

### DISCUSSION THREAD

**Discussion by muchenhen** [muchenhen/MuQianQiu#2] [Q&A] [answered]
似乎修掉了对手因为换牌行动多次的bug
增加了一个全局紧急退出按钮应对不可预知bug

<hr /><em>This discussion was created from the release <a href='https://github.com/muchenhen/MuQianQiu/releases/tag/v0.1.1-beta'>v0.1.1-beta</a>.</em>

**Comment by Alec-Ray** [comment, marked as answer]
(๑•̀ㅂ•́)و✧

**Reply by muchenhen** [reply]
o(￣▽￣)ｄ
